loadbalancer/algorithm_test.go

```diff
@@ -411,8 +412,67 @@ func TestConsistentHashBoundedLoadDistribution(t *testing.T) {
 	}
 }
 
+func TestConsistentHashKeyDistribution(t *testing.T) {
+	endpoints := []string{"http://abc.com:9001", "http://def.com9002", "http://ghi.com:9003", "http://jkl.com:9004", "http://mno.com:9005", "http://pqr.com:9006", "http://stu.com:9007", "http://vwx.com:9008", "http://yza.com:9009", "http://sjsjsjsj.com:9010"}
```

Comment:
Maybe something from "10.2.0.0/16" cidr like `"http://10.2.0.1:8080", "http://10.2.0.2:8080, ...`? 
I know it is tailoring to k8s but that feels more realistic then artificial random domains (i.e. we should face real distribution instead of randomizing endpoint names)